tests/gis_tests/test_geoforms.py

```diff
@@ -374,11 +374,21 @@ class PointForm(forms.Form):
 class GeometryWidgetTests(SimpleTestCase):
 
     def test_get_context_attrs(self):
-        """The Widget.get_context() attrs argument overrides self.attrs."""
+        # geom_type gets transfered in the returned context
+        widget = BaseGeometryWidget(attrs={'geom_type': 'POLYGON'})
+        context = widget.get_context('point', None, None)
```

Comment:
It seems odd to keep `'point'` here, even though it probably doesn't make much difference.
```suggestion
        context = widget.get_context('polygon', None, None)
```